core/lib/workarea/version.rb

```diff
@@ -1,11 +1,6 @@
 module Workarea
   module VERSION
-    MAJOR = 3
```

Comment:
I'm not sure why but just ditching these constants seems dangerous.

Reply:
We don't use the constants individually except in the github action to build documentation. I just updated that file to reference the new format. The goal here is to bolster the reliability of the gsubbing the version in the rake task.